resources/mediawiki.special/mediawiki.special.preferences.js

```diff
@@ -19,6 +19,18 @@ var $legends = $fieldsets.children( 'legend' )
 	.addClass( 'mainLegend' );
 
 	/**
+	 * Wikia Function - @see UC-206
```

Comment:
It looks like the indentation is off for these lines https://github.com/Wikia/app/blob/UC-206/resources/mediawiki.special/mediawiki.special.preferences.js#L21-L47